discovery-provider/src/queries/get_underground_trending.py

```diff
@@ -142,7 +132,7 @@ def get_scorable_track_data(session, redis_instance):
             Track.track_id.notin_(exclude_track_ids),
             Track.created_at >= (datetime.now() - timedelta(days=o)),
             follower_query.c.follower_count < S,
```

Comment:
Can we also replace follower_query here?

Reply:
If you look at the subquery, it uses follows within a time interval
`Follow.created_at < (datetime.now() - timedelta(days=f))`